plugins-mu/hm-stack/hm-stack.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Plugin Name: HM Stack Integration
+ * Description: Load the HM Stack API & Endpoint functionality.
+ * Version: 0.1.0
+ * Author: Human Made
+ * Author URI: http://hmn.md
+ *
+ * @package HMStackIntegration
+ */
+
+namespace HM_Stack;
+
+// Load files.
+require_once 'class-hm-stack-api.php';
```

Comment:
There should be an `inc` folder where the class files go, though see above comment about these maybe not needing to be classes.